frontend/src/routes/project/[project_code]/ChangeMemberRoleModal.svelte

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+	import Modal, { DialogResponse } from '$lib/components/modals/Modal.svelte';
+	import { lexSuperForm } from '$lib/forms';
+	import Form from '$lib/forms/Form.svelte';
+	import UserRoleSelect from '$lib/forms/UserRoleSelect.svelte';
+	import { ProjectRole } from '$lib/gql/graphql';
+	import t from '$lib/i18n';
+	import { z } from 'zod';
+	import { _changeProjectMemberRole } from './+page';
+
+	export let projectId: string;
+
+	let _name: string;
+
+	export async function open(member: { userId: string; name: string }): Promise<void> {
```

Comment:
good implementation of the modal. Man is it a pain when combined with a form, even I have a little trouble tracking it. I wonder if it would be better with a ModalForm that wraps that up so you can just use it like a modal? Just a thought I had, no action required.